spec/setup.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+const { EOL } = require('os');
+const { exec } = require('child_process');
+
+const envPath = path.join(__dirname, '..', '.env');
+let envData = '';
+
+const setWhitelistedAccount = () => {
+  // Load env
+  const data = fs.readFileSync(envPath, (err) => {
+    if (err) {
+      throw err;
+    }
+  });
+  envData = data.toString();
+
+  // Parse and remove comments
+  let envArray = data.toString().split(EOL);
+  envArray = envArray.filter((line) => {
+    if (line && !line.startsWith('#')) {
+      return line;
+    }
+  });
+
+  // check for whtelisted_account
+  const whitelist = envArray.find((line) =>
+    line.startsWith('WHITELISTED_ACCOUNTS')
```

Comment:
`WHITELISTED_ACCOUNTS`: Declare this as a constant and use everywhere.

Reply:
Done.